Userland/Libraries/LibC/stdio.cpp

```diff
@@ -657,6 +657,11 @@ char* fgets(char* buffer, int size, FILE* stream)
     return ok ? buffer : nullptr;
 }
 
+char* gets(char* buffer)
+{
+    return fgets(buffer, -1, stdout);
+}
+
```

Comment:
Why is this even needed?

Reply:
I needed this for a programm that I used for testing, that had questionable Copyrights attached though and failed the ci, so I removed the test, but left the patches

shall I make this another pull request?